packages/fxa-auth-server/test/scripts/must-reset.js

```diff
@@ -48,48 +49,47 @@ function createAccount(email, uid) {
 }
 
 const account1Mock = createAccount(
-  'user1@test.com',
-  'f9916686c226415abd06ae550f073cec'
+  `${Math.random() * 10000}@zmail.com`,
+  crypto.randomBytes(16).toString('hex')
 );
 const account2Mock = createAccount(
-  'user2@test.com',
-  'f9916686c226415abd06ae550f073ced'
+  `${Math.random() * 10000}@zmail.com`,
+  crypto.randomBytes(16).toString('hex')
 );
 
 const DB = require('../../lib/db')(config, log, Token, UnblockCode);
 
 describe('#integration - scripts/must-reset', async function () {
-  this.timeout(30000);
+  this.timeout(10000);
 
-  let db, server;
+  let db, oneEmailFilename, oneUidFilename, twoEmailsFilename, twoUidsFilename;
 
   before(async () => {
-    server = await TestServer.start(config);
     db = await DB.connect(config);
-    await db.deleteAccount(account1Mock);
-    await db.deleteAccount(account2Mock);
-  });
-
-  after(async () => {
-    await db.deleteAccount(account1Mock);
-    await db.deleteAccount(account2Mock);
-    return await TestServer.stop(server);
-  });
-
-  beforeEach(async () => {
     await db.createAccount(account1Mock);
     await db.createAccount(account2Mock);
-  });
 
-  afterEach(async () => {
-    await db.deleteAccount(account1Mock);
-    await db.deleteAccount(account2Mock);
+    const data = `${account1Mock.email}\n`;
+    oneEmailFilename = `./test/scripts/fixtures/${Math.random()}_one_email.txt`;
+    fs.writeFileSync(oneEmailFilename, data);
+
+    const data2 = `${account1Mock.uid}\n`;
+    oneUidFilename = `./test/scripts/fixtures/${Math.random()}_one_uid.txt`;
+    fs.writeFileSync(oneUidFilename, data2);
+
+    const data3 = `${account1Mock.email}\n${account2Mock.email}\n`;
+    twoEmailsFilename = `./test/scripts/fixtures/${Math.random()}_two_emails.txt`;
+    fs.writeFileSync(twoEmailsFilename, data3);
+
+    const data4 = `${account1Mock.uid}\n${account2Mock.uid}\n`;
+    twoUidsFilename = `./test/scripts/fixtures/${Math.random()}_two_uids.txt`;
+    fs.writeFileSync(twoUidsFilename, data4);
```

Comment:
Mentioned elsewhere, but do we want to clean up these files, or make sure they are written in a way that they are easily git ignored?

Reply:
This gets cleaned up now.